lib/SMTLIB2/Solver.cpp

```diff
@@ -252,7 +252,9 @@ class ProcessSMTLIBSolver : public SMTLIBSolver {
     }
     ::close(OutputFD);
 
-    int ExitCode = Prog(Args, InputPath, OutputPath, Timeout);
+    SmallString<64> ErrorPath("/dev/null");
+
+    int ExitCode = Prog(Args, InputPath, OutputPath, ErrorPath, Timeout);
```

Comment:
Can't this just be

    ..., OutputPath, /*ErrorPath=*/"/dev/null", Timeout);

?